operations/services.md

```diff
@@ -578,7 +578,7 @@ Updates price of a rate in the specified intervals. If the `CategoryId` is speci
 | --- | --- | --- | --- |
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
-| `RateId` | string | required | Unique identifier of the [Rate](services.md#rate) to update. |
+| `RateId` | string | required | Unique identifier of the base [Rate](services.md#rate) to update. |
```

Comment:
This would mean that you need to provide Id of the base rate if you need to update child rate. I'd reformulate this in the summary above this table that only rates that are not inheritted (that have no base rate).

Reply:
Thanks @siroky